packages/accounts/src/api/client.js

```diff
@@ -38,6 +39,7 @@ client.interceptors.response.use(
       }
       return Promise.reject(err);
     } else {
+      Sentry.captureException(err);
```

Comment:
Do we also need to capture the error for line 40?

Reply:
That was a mistake, please take another look at commit ae56bf9